content/en/blog/2022/2022-02-16-sig-node-ci-subproject-celebrates/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog
+title: 'SIG Node CI Subproject Celebrates Two Years of Test Improvements'
+date: 2022-02-16
+---
+
+
+**Authors:** Sergey Kanzhelev (Google), Elana Hashman (Red Hat)
+
+Ensuring the reliability of SIG Node upstream code is a continuous effort
+that takes a lot of behind-the-scenes effort from many contributors.
+There are frequent releases of Kubernetes, base operating systems,
+container runtimes, and test infrastructure that result in a complex matrix that
+requires attention and steady investment to "keep the lights on."
+Since its organization in May 2020, the SIG Node CI subproject has run a weekly
+meeting, and even the full hour is often not enough to complete triage
+of all bugs, PRs, and issues and discuss all related ongoing work within the SIG.
```

Comment:
Maybe expand some abbreviations (CI and SIG):
```suggestion
In May 2020, the Kubernetes node special interest group ("SIG Node") organized a new
subproject for continuous integration for node-related code changes. Since its
inauguration, the SIG Node CI subproject has run a weekly meeting, and even the full hour
is often not enough to complete triage of all bugs, PRs, and issues and discuss all
related ongoing work within the SIG.
```
?

Reply:
```suggestion
In May 2020, the Kubernetes node special interest group ("SIG Node") organized a new
subproject for continuous integration for node-related code changes. Since its
inauguration, the SIG Node CI subproject has run a weekly meeting, and even the full hour
is often not enough to complete triage of all bugs, test related PRs and issues, and discuss all
related ongoing work within the subgroup.
```